mintapi/signIn.py

```diff
@@ -523,6 +540,10 @@ def search_mfa_method(driver):
     for method in MFA_METHODS:
         mfa_token_input = mfa_token_button = mfa_method = span_text = result = None
         try:
+            mfa_token_select = driver.find_element_by_css_selector(
```

Comment:
@tsdg112 this section won't work.  This assumes that every user has the "select" page, which they will not.  In fact, this is the core of the "dynamic MFA flow" that we built.